src/XCCDF_POLICY/xccdf_policy.c

```diff
@@ -981,6 +979,52 @@ _xccdf_policy_rule_evaluate(struct xccdf_policy * policy, const struct xccdf_rul
 		return _xccdf_policy_report_rule_result(policy, result, rule, NULL, XCCDF_RESULT_NOT_SELECTED, NULL);
 	}
 
+	bool all_requires_met = true;
```

Comment:
I would assume we do requires as the last step of the resolve. If the spec goes against us there is nothing we can do but it's a shame.